src/build.d

```diff
@@ -171,13 +166,10 @@ A dependency will be skipped if all targets are older than all sources.
 This script is by default part of the sources and thus any change to the build script,
 will trigger a full rebuild.
 
-The function buildDMD defines the build order of its dependencies.
 */
 
 /// Returns: the dependency that builds the lexer
-alias lexer = memoize!defineLexer;
-auto defineLexer()
-{
+alias lexer = memoize!(function() {
```

Comment:
I've found a way to define dependencies without having to create a "defineXXX" function by making use of an anonymous function here.  It saves a couple lines and now we don't have an extra function around that we don't want other people to use.